readme.md

```diff
@@ -2,7 +2,7 @@
 
 ![hqkkb](https://user-images.githubusercontent.com/173738/71843051-51f52e00-30c3-11ea-9bc8-2feae2ac355f.jpeg)
 
-This is a newb attempt at tuning NPKC's HHKB firmware to better reflect the original PFU HHKB2 layout.
```

Comment:
Looks like feature, not :bug: :)

Reply:
Good catch @xandkar and thanks @JStrebeyko :)

> 
> newb
> /njuːb/
> nouninformal•US
> noun: newb; plural noun: newbs

   >  short for newbie.
   > "it's easy enough for total newbs to pick up and play"